interface/forms/vitals/C_FormVitals.class.php

```diff
@@ -21,7 +21,7 @@ class C_FormVitals extends Controller {
 
     function __construct($template_mod = "general") {
         parent::__construct();
-        $returnurl = $GLOBALS['concurrent_layout'] ? 'encounter_top.php' : 'patient_encounter.php';
+        $returnurl = 'encounter_top.php';
```

Comment:
And here.
Just four lines of real changes, just the $returnurl that is getting fixed, right?